oak/module/defines.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef OAK_MODULE_DEFINES_H_
+#define OAK_MODULE_DEFINES_H_
+
+#define export_to_wasm extern "C" __attribute__((visibility("default")))
```

Comment:
How about `wasm_export` and `wasm_import(module_name)`?

Reply:
Done.. sorry for two emails, I couldn't find this comment in the review :(